app/components/Grid.tsx

```diff
@@ -134,7 +139,7 @@ export const GridView = ({
         gridWidth={grid.width}
         gridHeight={grid.height}
         active={isActive}
-        entryCell={entryCells.some((p) => cellIndex(grid, p) === idx)}
+        entryCell={entryCell}
```

Comment:
No big deal but I have a slight preference for undoing the change that shifted entryCell into it's own var - just to keep this PR's surface area as small as possible. I wonder if you had used it in one of your other iterations so needed to pull it out for that